docs/appkit/javascript/core/installation.mdx

```diff
@@ -57,6 +60,13 @@ npm install @reown/appkit @reown/appkit-adapter-ethers ethers
 npm install @reown/appkit @reown/appkit-adapter-solana
 ```
 
+</PlatformTabItem>
+<PlatformTabItem value="bitcoin">
+
+```bash npm2yarn
+npm install @reown/appkit @reown/appkit-adapter-bitcoin bitcoinjs-lib
```

Comment:
I think we don't need to have `bitcoinjs-lib` installed by the users.
```suggestion
npm install @reown/appkit @reown/appkit-adapter-bitcoin
```
This repeats for all the install examples

Reply:
oks .. you are using it in one example ... i can delete ... np